### PR TITLE
Update python notebook for docs

### PR DESCRIPTION
**Pull request by mlathara**
Update docs notebook to use protobuf. And some changes that @Santhosh-K-M mentioned while reading the docs...

Edit: it is linked out from the checks below but just for completeness here is the link to what the docs look like after these changes: https://genomicsdb--269.org.readthedocs.build/en/269/

### CODE REVIEW

File: docs/cli-tools.rst [hunk 1]
```diff
@@ -49,6 +49,100 @@ gt_mpi_gather
...
+
+  Most of the fields are self-explanatory (take a look at the :ref:`terminology section <Concepts + Terminology>` section).
+  The following fields are mandatory:
+  
+  * workspace (type: string or list of strings)
+  * array (type: string or list of strings)
+  * query_column_ranges : This field contains is a list of lists. Each member list contains the column ranges that are to be queried. Each element of the inner list can be a single integer, representing the single column position to be queried, or a list of size 2 representing the column range to be queried.
+
+    In the above example, the process will query column range \[0-100\] (inclusive) and the position 500 and return all *VariantCalls* intersecting with these query ranges/positions.
+    This field is a list of lists (and other fields are lists of strings), in order to support using GenomicsDB with MPI. Each member of the othermost list can be processed by a separate MPI rank.
```
Comment (webberlh) [inline docs/cli-tools.rst:74]: It looks like this paragraph was reworded to remove the reference to the wiki, but could we just drop it entirely? I may be missing a larger context, but at least in my mind this seems more like an implementation detail than something a docs consumer would need to know.

**Review by webberlh — APPROVED** [review]
LGTM, my only comment is more a matter of style than anything so whichever way it gets resolved this seems good to me.

**Review by nalinigans — DISMISSED** [review]

File: docs/cli-tools.rst (hunk 1 above)
Comment (mlathara) [reply docs/cli-tools.rst:74]: Good point that I don't explicitly call out why this is really relevant to the user. The thing that is relevant for a docs consumer/user is that unless MPI is being used, only the first element of the outermost list is processed. I'll add wording to that effect.

**Review by webberlh — APPROVED** [review]
LGTM!

**Pull request merged**